apps/api/package.json

```diff
@@ -15,11 +15,11 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@cryptochords/shared": "*",
     "@typescript-eslint/eslint-plugin": "^6.20.0",
     "dotenv": "^16.3.1",
     "express": "^4.18.2",
-    "web3": "^4.3.0",
-    "@cryptochords/shared": "*"
+    "web3": "^4.3.0"
```

Comment:
To prevent problems, let's fix all the versions the API (and app) uses. In some scenarios the lock-file is not enough. See:

https://github.com/hemilabs/software-development-process/blob/master/guides/JavaScript%20Dependencies.md

Reply:
Done.